libraries/Microsoft.Bot.Builder/BotFrameworkAdapter.cs

```diff
@@ -1202,7 +1257,7 @@ protected virtual async Task<AppCredentials> BuildCredentialsAsync(string appId,
         /// <param name="cancellationToken">Cancellation token.</param>
         /// <returns>ConnectorClient instance.</returns>
         /// <exception cref="NotSupportedException">ClaimsIdentity cannot be null. Pass Anonymous ClaimsIdentity if authentication is turned off.</exception>
-        private async Task<IConnectorClient> CreateConnectorClientAsync(string serviceUrl, ClaimsIdentity claimsIdentity, CancellationToken cancellationToken)
+        private async Task<IConnectorClient> CreateConnectorClientAsync(string serviceUrl, ClaimsIdentity claimsIdentity, string audience = null, CancellationToken cancellationToken = default)
```

Comment:
given this is private it might be tempting to just always have this required - basically makes the call site more explicit